clang/lib/Parse/ParseExpr.cpp

```diff
@@ -2218,27 +2218,20 @@ Parser::ParsePostfixExpressionSuffix(ExprResult LHS) {
         CalledSignatureHelp = true;
         return PreferredType;
       };
+      bool KnownInvalidCall = false;
```

Comment:
```suggestion
      bool ExpressionListIsInvalid = false;
```

(Might not be a call)